packages/react-components/react-nav-preview/src/components/sharedNavStyles.styles.ts

```diff
@@ -23,7 +23,7 @@ export const useRootDefaultClassName = makeResetStyles({
   position: 'relative',
   justifyContent: 'start',
   gap: tokens.spacingVerticalL,
-  padding: tokens.spacingVerticalMNudge,
+  padding: `${tokens.spacingVerticalMNudge} ${tokens.spacingVerticalS} ${tokens.spacingVerticalMNudge} ${tokens.spacingVerticalS}`,
```

Comment:
Would it make sense here & below to use horizontal tokens for the left/right padding?

Reply:
Good catch - thanks!